packages/meditrak-app/package.json

```diff
@@ -1,107 +1,85 @@
+    "@react-native-async-storage/async-storage": "^1.19.3",
+    "@react-native-camera-roll/camera-roll": "^6.0.0",
+    "@react-native-community/datetimepicker": "^7.6.1",
+    "@react-native-community/geolocation": "^3.1.0",
+    "@react-native-community/netinfo": "^9.4.1",
+    "@tupaia/access-policy": "workspace:^",
+    "@tupaia/expression-parser": "workspace:^",
+    "appcenter": "^5.0.0",
+    "appcenter-analytics": "^5.0.0",
+    "bson-objectid": "^2.0.4",
+    "moment": "^2.29.4",
     "nanoid": "^2.0.3",
-    "npm-bump": "^0.0.23",
-    "prop-types": "^15.7.2",
-    "react": "16.13.1",
-    "react-native": "^0.63.5",
-    "react-native-camera": "^4.2.1",
-    "react-native-database": "^0.1.7",
-    "react-native-device-info": "^7.2.1",
-    "react-native-document-picker": "^8.2.1",
-    "react-native-dotenv": "^0.2.0",
-    "react-native-file-access": "^1.7.1",
-    "react-native-fs": "^2.16.6",
-    "react-native-gesture-handler": "~1.7.0",
-    "react-native-image-picker": "https://github.com/beyondessential/react-native-image-picker.git#fix-v2-android-13",
-    "react-native-linear-gradient": "^2.5.6",
-    "react-native-localize": "^2.0.1",
+    "react": "18.2.0",
+    "react-native": "0.72.5",
+    "react-native-config": "^1.5.1",
+    "react-native-database": "^0.3.3",
+    "react-native-device-info": "^10.9.0",
+    "react-native-document-picker": "^9.0.1",
+    "react-native-fs": "^2.20.0",
+    "react-native-gesture-handler": "^2.13.0",
+    "react-native-image-picker": "^7.0.0",
+    "react-native-linear-gradient": "^2.8.3",
+    "react-native-localize": "^3.0.2",
     "react-native-markdown-renderer": "^3.2.8",
     "react-native-menu": "^0.23.0",
-    "react-native-modal-datetime-picker": "^9.1.0",
-    "react-native-modalbox": "^2.0.0",
-    "react-native-permissions": "^3.6.1",
-    "react-native-qrcode-scanner": "^1.5.5",
-    "react-native-qrcode-svg": "^6.1.2",
-    "react-native-reanimated": "~1.13.0",
-    "react-native-safe-area-context": "3.1.4",
-    "react-native-screens": "~2.10.1",
-    "react-native-share": "^7.9.1",
+    "react-native-modal-datetime-picker": "^17.1.0",
+    "react-native-qrcode-svg": "^6.2.0",
+    "react-native-safe-area-context": "^4.7.2",
+    "react-native-share": "^9.4.1",
     "react-native-side-menu": "^1.1.3",
-    "react-native-svg": "^12.1.0",
-    "react-native-vector-icons": "^7.1.0",
-    "react-native-webview": "^11.23.1",
-    "react-navigation": "^4.4.0",
+    "react-native-svg": "^13.13.0",
+    "react-native-vector-icons": "^10.0.0",
+    "react-native-vision-camera": "^3.4.1",
+    "react-native-webview": "^13.6.0",
+    "react-navigation": "^4.4.4",
     "react-navigation-redux-helpers": "^4.0.1",
-    "react-navigation-stack": "^2.8.2",
-    "react-qr-code": "^2.0.8",
-    "react-redux": "^5.0.7",
-    "realm": "^6.1.5",
-    "redux": "^3.7.2",
-    "redux-persist": "^5.9.1",
+    "react-navigation-stack": "^2.10.4",
+    "react-redux": "^8.1.2",
+    "realm": "^12.1.0",
```

Comment:
When testing this locally I was having issues with a missing `realm-constants.json` file in `node_modules/ream`. There seems be something that some other users have encountered: https://github.com/realm/realm-js/issues/6270

Apparently there's meant to be a post-install script that creates that file but something seemed to be going wrong for me.

Bumping to `12.3.0` fixed the issue though, currently `yarn.lock` has resolved to `12.2.1`

Reply:
Try to upgrade to v12.3.1 as we have a fix for it :smile:

Reply:
Thanks @kneth :pray: